n3fit/src/n3fit/tests/test_layers.py

```diff
@@ -197,12 +203,12 @@ def test_rotation_flavour():
     # Apply the rotation using numpy tensordot
     x = np.ones(8)  # Vector in the flavour basis v_i
     x = np.expand_dims(x, axis=[0, 1])  # Give to the input the shape (1,1,8)
-    mat = fitbasis_to_NN31IC(flav_info, 'FLAVOUR')  # Rotation matrix R_ij, i=flavour, j=evolution
+    mat = fitbasis_to_NN31IC(flav_info, "FLAVOUR")  # Rotation matrix R_ij, i=flavour, j=evolution
```

Comment:
FWIW I'd be a lot happier if the `-S` option was default in black. Not only do I find these diffs a bit distracting in the context of a complicated PR but also don't think it is an improvement in any way.

Reply:
i thinl going for "whatever is default in black" is a good idea because it can be done mindlessly I can just tell people to install it and use it.

it's is true that for files like this the reviewer pays the prize of my past sloppyness. I'll undo some of them before marking this for review (I'm adamant on the `fkparser.py` though because it's other people's fault :P)

Reply:
I was not going to complain until some threshold was surpassed...